_ont/ont-fs-com-gpon-onu-stick-with-mac.md

```diff
@@ -112,6 +112,23 @@ cp /etc/mibs/data_1g_8q.ini /etc/mibs/data_1g_8q.ini.bak
 sed 's/256 0 HWTC 0000000000000/256 0 YOUR_VENDOR_ID YOUR_ONU_VERSION/' -i /etc/mibs/data_1g_8q.ini
 ```
 
+## Enable `data_1g_8q_us1280_ds512` OMCI MIB file
```

Comment:
for the homogeneity, the dot ini (`.ini`) should also be put here.

i suggest also putting that this enablement is useful for 2500 Mbps speed profiles in the title